src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LeftShiftInstruction.cs

```diff
@@ -29,9 +31,9 @@ public override int Run(InterpretedFrame frame)
                 }
                 else
                 {
-                    frame.Push((sbyte)(((sbyte)value) << ((int)shift)));
+                    frame.Push((sbyte)((sbyte)value << (int)shift));
```

Comment:
Nice. Some might find they and the redundant casts improve readability, but I for one find the opposite.